airflow/sentry.py

```diff
@@ -149,6 +154,10 @@ def wrapper(task_instance, *args, session=None, **kwargs):
 
         return wrapper
 
+    def flush(self):
+        from sentry_sdk import Hub
+        Hub.current.flush()
```

Comment:
Should we be calling https://github.com/getsentry/sentry-python/blob/68ddcf459078f892ee25fbc354c9044057fbd0ef/sentry_sdk/api.py#L221-L229 instead?

Reply:
Looks like that was added in 0.7.3 and we already depend on  >=0.8.0

Reply:
Good point, I'll update - thanks.

Reply:
I think this one still pending @mikeclarke if you dont mind :-)

Reply:
yep will update now, thanks.